composer.json

```diff
@@ -39,7 +39,8 @@
     "autoload": {
         "psr-4": {
             "EzSystems\\PlatformHttpCacheBundle\\": "src",
-            "EzSystems\\PlatformHttpCacheBundle\\Tests\\": "tests"
+            "EzSystems\\PlatformHttpCacheBundle\\Tests\\": "tests",
+            "Ibexa\\HttpCache\\": "src"
```

Comment:
On merge up into 4.0 please move it as the first entry. Right now since most of the classes have old ns it wouldn't make sense.